doc/builtins.md

```diff
@@ -1462,6 +1463,37 @@ The `default` value is returned if the input `value` is null or an empty string
 Yo
 ```
 
+[:top:](#builtins)
```

Comment:
Just to make sure, you did generate this by running the `Scriban.DocGen` program right?

Reply:
Yep

Reply:
It also generated docs for `date.now` but I removed it because it's not there in `master` branch. Also it makes the docs tests fail because it expects year 2017 in the output. I decided not to fix it since it's not related to this PR.

Reply:
ok, yep, will check for date.now